docs/integrate_gcp_using_supported_system.md

```diff
@@ -0,0 +1,176 @@
+2. Search for the service account then add permissions for **Organization Administrator** and **Logs Configuration Writer**
+3. Click **SAVE**
+
+![Service Account Org Permissions](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_service_account_org_permissions_1024px.gif)
+
+#### Create Service Account Key
+This section covers creating a service key and downloading to the local system as a `JSON` file
+
+![Create Key](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_create_key_1024px.gif)
+
+1. Login to the Google Cloud Console
+2. Select the Project created for Lacework Resources
+3. Click on the **Navigation Menu** and select **IAM & Admin** -> **Service Accounts**
+4. Click on the Actions menu next to the service account
+5. Click **Create Key**
+6. Select JSON for the format of the key
+7. Click **Create** to download the key locally
+
+### Run Terraform to integrate GCP at Organization Level
+This section covers executing Terraform. It is recommended you use a developer's code editor such as [VSCode](https://code.visualstudio.com/download), [Atom](https://atom.io/), or [Sublime](https://www.sublimetext.com/3). You will also need a terminal to run `terraform` commands such as `bash`, `zsh`, `powershell` or `cmd`
+![Terraform Code](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_org_terraform_init_1024px.gif)
+1. Open an EDITOR of choice and create a new file called `main.tf`
+2. Copy/paste the following code into the `main.tf` and save the file
+```hcl
+provider "google" {
+  credentials = file("account.json")
+  project     = "my-project-id"
+}
+
+provider "lacework" {}
+
+module "gcp_organization_config" {
+	source          = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/config?ref=master"
+	org_integration = true
+	organization_id = "my-organization-id"
+}
+
+module "gcp_organization_audit_log" {
+	source                       = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/audit_log?ref=master"
+	bucket_force_destroy         = true
+	org_integration              = true
+	use_existing_service_account = true
+	service_account_name         = module.gcp_organization_config.service_account_name
+	service_account_private_key  = module.gcp_organization_config.service_account_private_key
+	organization_id              = "my-organization-id"
+}
+```
+
+3. Update both the `credentials` and the `project` in the `provider "google"` to reference your credentials file downloaded in the previous section, and the Google Project ID
+4. Open a terminal, change directory to the location where you have saved the `main.tf` and run the command `terraform init` to initialize the project
+5. Run `terraform plan` to review the changes, and then run `terraform apply` when you are ready to apply the changes
```

Comment:
I didn't see a section to install and configure the Lacework CLI, we need that before running terraform commands